content/coop.html

```diff
@@ -244,3 +244,48 @@ <h3> Admission average </h3>
   <span class="graph-title"> Admission average compared to monthly salary</span>
   <div id="admission-salary"></div>
 </div>
+
+<div class="vbx2"></div>
+<h3> Interviews </h3>
+<p class="full">
+  Over the coop interview cycles students sometimes missed an interview or were late to an interview. 
+</p>
+<div class="vb"></div>
+<div class="half">
+  <span class="graph-title">Missed an Interview</span>
+  <div id="missed_interview" class="graph-center"></div>
+</div>
+<div class="half second">
+    <span class="graph-title">Late to an Interview</span>
+    <div id="late_interview" class="graph-center"></div>
+</div>
+<div class="vb"></div>
+<div class="half">
```

Comment:
loved the stories :D there's also a `path-section` class that was used in misc -> "beauty in the details" section that has some sort of styling for stories, if you wanted to take a look

Reply:
the below comments are small nits to reading fluidity